src/FixedDataTable.js

```diff
@@ -1096,12 +1132,12 @@ class FixedDataTable extends React.Component {
 
     if (scrollJumpedX) {
       scrollActions.jumpScrollX();
-      onHorizontalScroll && onHorizontalScroll(scrollX);
+      this._scrollToX(scrollX);
```

Comment:
A jump scroll is detected happens `after` the actual scroll happened (because it's instantaneous). Wouldn't this be just making a scroll call again? (Same down below)

Reply:
`scrollActions.jumpScrollX()` made me think it is triggering scroll, reverted these changes now that I know we are just triggering callbacks in this function

Reply:
Apologies, we need the call to `this._scrollToX`. Your initial change was correct.

Earlier we didn't need a scroll call since scrolling was controlled. 
Here since it's uncontrolled, whenever a scroll happens outside of the scroll bar (which is exactly what jump scrolls are), we need to explicitly set the scroll.

Also, this shouldn't lead to extra renders since `this._onVerticalScroll` and `this._onHorizontalScroll` checks for value change before issuing scroll actions (which was my initial concern).

Reply:
ok, I will get these changes back in that case.